linkedin-clone/src/actions/index.js

```diff
@@ -1083,4 +1093,32 @@ export function declineGroupJoinRequest(request) {
 
   const pagesCollectionRef = collection(db, 'Pages');
   await addDoc(pagesCollectionRef, newPage); 
-};
\ No newline at end of file
+};
+
+export async function deleteProfile() {
```

Comment:
Can the user reopen the account with the same email?

Reply:
good point

Reply:
problem resolved